modules/swagger-codegen/src/main/java/io/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -106,6 +107,8 @@ protected Object resolvePropertyToExample(String mediaType, Property property, S
             return 1.23f;
         } else if (property instanceof IntegerProperty) {
             return 123;
+        } else if (property.getClass().equals(BaseIntegerProperty.class)) {
```

Comment:
@fultonm eventually we want to make `toDefaultValue` abstract so that all language generators must implement it as the default might not work for the language. Before that, we'll need to add `toDefaultValue` in each language generator, which will probably take quite amount of work as the project supports many generators.